CHANGELOG.md

```diff
@@ -2,12 +2,13 @@
 
 This changelog is a work in progress and may contain notes for versions which have not actually been released. Check the [Releases](https://github.com/0xProject/0x-mesh/releases) page to see full release notes and more information about the latest released versions.
 
-## v5.0.1-beta-0xv3
+## v5.0.1-beta-0xv3.1
```

Comment:
@fabioberger What's the reasoning here? `0xv3` is meant to indicate compatibility with 0x protocol version 3. This change makes it look like we are targeting protocol version 3.1.

Reply:
Ah shoot, you are right. I was treating `beta-0xv3` as the beta tag. `.1` being the next version of that beta.

Let's go ahead and version them separately.